tests/test_scripts/test_doppler_mcp_bootstrap.py

```
@@ -0,0 +1,133 @@
+    path = REPO_ROOT / "configs" / "doppler" / "mcp_secrets_manifest.json"
+    parsed = json.loads(path.read_text(encoding="utf-8"))
+    assert isinstance(parsed, dict)
+
+
+def test_cmd_status_missing_required(monkeypatch: pytest.MonkeyPatch) -> None:
+    mod = _load_bootstrap()
+    monkeypatch.setattr(mod, "cmd_verify", lambda _a: 0)
+    monkeypatch.setattr(
+        mod,
+        "doppler_run",
+        lambda *a, **k: MagicMock(returncode=0, stdout="{}"),
```

Ruff (0.15.7)

[warning] Unused lambda argument: `a`

(ARG005)